pkg/controllers/networkpolicy/controller.go

```diff
@@ -731,13 +733,31 @@ func (r *NetworkPolicyReconciler) SetupWithManager(mgr ctrl.Manager) error {
 	if mgr == nil {
 		return nil
 	}
-	r.antreaClient = antreanetworkingclient.NewForConfigOrDie(mgr.GetConfig())
+
+	r.createAntreaClient(mgr, antreaKubeconfig)
```

Comment:
imo its cleaner to let `createAntreaClient` return the client then set r.antreaClient here like before. This way the func also can be a normal func instead of member func of `NetworkPolicyReconciler`.